ui/src/components/form/docker_image_combo_box/DockerRegistryPopover.js

```diff
@@ -28,7 +28,7 @@ export const DockerRegistryPopover = ({ value, registryOptions, onChange }) => {
           iconType="arrowDown"
           iconSide="right"
           onClick={togglePopover}>
-          {registryOptions.find((o) => o.value === value).inputDisplay}
+          {(registryOptions.find((o) => o.value === value) || {}).inputDisplay}
```

Comment:
Error handling in case some Docker registries used in existing configs were removed.

Reply:
No longer a valid prop, this throws a console error if set: https://elastic.github.io/eui/#/navigation/context-menu